assets/i18n/active.fr.json

```diff
@@ -48,8 +48,8 @@
     "other": "Les paramètres du sondage permettent une personnalisation plus poussée, e.g. `/{{.Trigger}} \"Question\" \"Réponse 1\" \"Réponse 2\" \"Réponse 3\" --progress --anonymous`. Les éléments de configuration disponibles sont:"
   },
   "command.help.text.pollSetting.multi-vote": {
-    "hash": "sha1-aadf6806b8899a61f1e79282e0f981f30b369634",
-    "other": "Autoriser les utilisateurs à voter pour X options"
+    "hash": "sha1-8848f6b6a03f323a14609708897824ad178d2dd8",
+    "other": "Allow users to vote for X options. Default is 1. If X is 0, users have an unlimited amount of votes."
```

Comment:
I'm confused. Shouldn't these translation strings be in France? Or am I missing something?

Reply:
What the… I don’t understand, I translated all the changes. Something must have failed in my workflow..

Give me a few minutes to fix it.

Reply:
Here, fixed.